pkg/apis/camel/v1/integrationplatform_types_support.go

```diff
@@ -178,13 +179,17 @@ func (in *IntegrationPlatformStatus) RemoveCondition(condType IntegrationPlatfor
 	in.Conditions = newConditions
 }
 
-// IsKanikoCacheEnabled tells if the KanikoCache is enabled on the integration platform build spec
-func (b IntegrationPlatformBuildSpec) IsKanikoCacheEnabled() bool {
-	if b.KanikoBuildCache == nil {
-		// Cache is disabled by default
-		return false
+// IsOptionEnabled tells if the KanikoCache is enabled on the integration platform build spec
```

Comment:
Still reference to Kaniko. As it now is a general purpose we should change the comment as well